frontend/src/components/pages/details/components/DetailsDownloadIcons/DetailsDownloadIcons.tsx

```diff
@@ -84,36 +87,46 @@ export const DetailsDownloadIcons: React.FC<DetailsTopIconsProps> = ({
 
       <div className="flex space-x-4">
         {details.pdfUri && (
-          <DetailsButton url={details.pdfUri}>
-            <Printer size={30} />
-          </DetailsButton>
+          <ToolTip toolTipText={intl.formatMessage({ id: 'tooltip.print' })}>
+            <DetailsButton url={details.pdfUri}>
+              <Printer size={30} />
+            </DetailsButton>
+          </ToolTip>
         )}
 
         {dropdownButtonOptions.length > 0 && (
-          <DetailsButtonDropdown options={dropdownButtonOptions}>
-            <Download className="text-primary1" size={size} />
-          </DetailsButtonDropdown>
+          <ToolTip toolTipText={intl.formatMessage({ id: 'tooltip.download' })}>
+            <DetailsButtonDropdown options={dropdownButtonOptions}>
+              <Download className="text-primary1" size={size} />
+            </DetailsButtonDropdown>
+          </ToolTip>
         )}
 
         {Number(details.id) && !hideReport && getGlobalConfig().enableReport && (
-          <DetailsButton url="#details_report" onClick={() => setReportVisibility(true)}>
-            <AlertTriangle size={size} />
-          </DetailsButton>
+          <ToolTip toolTipText={intl.formatMessage({ id: 'tooltip.report' })}>
```

Comment:
I don't know if grouping all tooltip locales in the `tooltip` scope is the best use. For example with this line, maybe it is better to put it in the `report` scope?
What do you think @camillemonchicourt?

Reply:
Yes indeed.